__tests__/app.test.js

```diff
@@ -20,4 +25,33 @@ describe.only("GET /api", () => {
         expect(endpoints).toEqual(endpointsJson);
       });
   });
+
+  describe("GET /api/topics", () => {
+    test("200: Responds with an array of all topics", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const { topics } = body;
+          expect(topics.length).toBeGreaterThan(0);
```

Comment:
well done for remembering to have a length check to prevent false passes from the foreach!